src/app/core/entity/default-field-value/handle-default-values.usecase.ts

```diff
@@ -93,6 +93,10 @@ export class HandleDefaultValuesUseCase {
         return;
       }
 
+      if (change.length != null && change.length === 1) {
+        change = change[0];
+      }
+
```

Comment:
can you add a comment here, what the reason behind this is?
Seems counter-intuitive to me. Doesn't that potentially break some cases of "isArray" type fields?

Reply:
Good finding, i added two more tests and a code comment for clarification.